guides/release/components/built-in-components.md

```diff
@@ -1,43 +1,42 @@
-Out of the box, Ember provides 2 components for building a form:
+Ember fournit deux composants prêts à l'emploi pour construire des formulaires:
 
 * [`<Input>`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Input?anchor=Input)
 * [`<Textarea>`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
 
-These components are similar in HTML markup to the native `<input>` or `<textarea>` elements. In contrast to the native elements, `<Input>` and `<Textarea>` automatically update the state of their bound values.
-
+Le markup HTML de ces composants est similaire à celui des éléments natifs `<input>` et `<textarea>`. À la différence des éléments natifs, `<Input>` et `<Textarea>` mettent automatiquement à jour l'état des valeurs qui leur sont liées.
 
 ## `<Input>`
 
-We mentioned that the built-in components are similar in HTML markup to their native counterparts. What does this mean?
+Nous avons dit plus haut que les composants _built-in_ (c'est-à-dire "construits à l'intérieur du framework") ont un markup HTML similaire à celui de leur équivalent natif. Que'est-ce que ça signifie ?
```

Comment:
Qu'est-ce que ça signifie ? - Mistyping of _Qu'est-ce_

Reply:
```suggestion
Nous avons dit plus haut que les composants _built-in_ (c'est-à-dire "construits à l'intérieur du framework") ont un markup HTML similaire à celui de leur équivalent natif. Qu'est-ce que ça signifie ?
```